### DISCUSSION TITLE
GoPro 11 Mini & Schedule Startrails Mode

### DISCUSSION THREAD

**Discussion by ZontiSenior** [gopro/labs#435] [Q&A] [answered]
Hello
My question would be, with Gopro Labs, on a GoPro 11 mini, can I have the Startrails, Light Paint, Vehicle Lights modes start from a scheduled time?
If so, would anyone have an example.
Thanks for your help
Gerhard

**Comment by dnewman-gpsw** [comment, marked as answer]
Yes it can do this. Procedure:

1) On camera: set the properties for your long exposure mode, like Star Trail, resolution, aspect ratio, trail length and shutter length.  Not all of these configurations are settable via QR Code (something for the future.)
2) QR Code: Set the time and date using https://gopro.github.io/labs/control/precisiontime/  If your camera's time is wrong, it will wake at the wrong time.  (optional if you know the camera's time is correct)
4) Your final QR command will be in this form !HH:MMNQmXX!S  - replace HH:MM with your 24hour clock start time. So for 11:15pm, use 23:15.  Replace XX with the mode you want: ST  - mode Star Trails, LP - Light Painting or LT - Vehicle Lights (or light trails).  If the camera is already in the correct mode, you can skip the mXX section, but it is there for thoroughness. 

Example for 11pm Star Trails https://gopro.github.io/labs/control/set/?cmd=!23:00NQmST!S

The !23:00NQ - Action sleep the camera until 11pm, wake, then NQ - Nothing Quickly.  You can do !23:00SQ if you know the camera is in the correct mode, SQ - Start Quick.  The Quick part explained: sometimes the camera boots 30-40s early, without the Q the camera will wait, but if you have 15s shutdown (like Mini's default), the camera might shutdown before the next command starts (so use the Q.)  mST - mode set Star Trails,  !S - Action Start.

**Reply by ZontiSenior** [reply]
Thank you very much for this competent help. It fulfils my wishes exactly!
Many greetings from Wels in Austria
Gerhard
